python/utils.py

```diff
@@ -279,16 +279,24 @@ def override_build_config_from_env(config):
 
 
 class build_ext(setuptools.command.build_ext.build_ext):
+    def __init__(self, *args, **kwargs):
+        self.install_dir = kwargs.pop("install_dir", None)
+        super().__init__(*args, **kwargs)
+
     def build_extension(self, ext):
         if ext.name == "nvfuser._C":
             # Copy files on necessity.
             filename = self.get_ext_filename(self.get_ext_fullname(ext.name))
             fileext = os.path.splitext(filename)[1]
 
             libnvfuser_path = os.path.join(
-                "./nvfuser_common/lib", f"libnvfuser{fileext}"
+                os.path.join(self.install_dir, "lib"), f"libnvfuser{fileext}"
```

Comment:
If `install_dir` is not specified, then it expects the library to be in `nvfuser_common`.
Something like this should work:
```python
    install_prefix = (
        os.path.join(cwd, "nvfuser_common")
        if not config.install_dir
        else config.install_dir
    )
    libnvfuser_path = os.path.join(
        os.path.join(install_prefix, "lib"), f"libnvfuser{fileext}"
```

Reply:
right! I fixed that. Since the same snippet you proposed is also used in the `cmake` function, I factored it out as a separate function. Let me know if you prefer having it duplicated.

Reply:
I appreciate the refactor. 👍🏼